src/QsCompiler/DataStructures/Diagnostics.fs

```diff
@@ -896,6 +900,9 @@ type DiagnosticItem =
             | WarningCode.UseOfFutureReservedKeyword -> "The symbol will be reserved for internal use in the future."
             | WarningCode.UseOfUnderscorePattern ->
                 "Double underscores as well as underscores before a dot or at the end of a namespace name will be reserved for internal use in the future."
+            | WarningCode.DeprecatedTupleBrackets ->
+                "Deprecated syntax. Parentheses here are no longer required and will not be supported in the future."
```

Comment:
Hm, I think this would be a good place to also let people know that a non-block version of the statement exists. Maybe add something like "Consider using a simply use-statement" (C# has a similar message - I'd look at their phrasing there and use that).

Reply:
There's no warning specifically for the block form; the extra parentheses and deprecated keyword warnings are more general than that. At the moment, the only deprecated keywords are `using` and `borrowing`, so I could repurpose that warning and make it specifically about the `using` or `borrowing` blocks.

But, it also feels like this is really a different kind of warning, unrelated to the keyword change. Like a "unnecessary qubit allocation block" warning that could detect `use`/`borrow` blocks that are the last statement of a scope...

Reply:
I updated the warning message to mention the non-block version.